src/app/clusters/operational-state-server/operational-state-server.h

```diff
@@ -262,14 +267,14 @@ class Delegate
     virtual CHIP_ERROR GetOperationalStateAtIndex(size_t index, GenericOperationalState & operationalState) = 0;
 
     /**
-     * Fills in the provided GenericOperationalPhase with the phase at index `index` if there is one,
+     * Fills in the provided MutableCharSpan with the phase at index `index` if there is one,
```

Comment:
This needs to document how a callee communicates "PhaseList is null".

Also: is there any use case for "empty but not null" for PhaseList?  Looks like not, but just checking.

Reply:
Optimize some document, please help check if this can match the use case for "empty but not null" for PhaseList